rules/consistent-destructuring.js

```diff
@@ -98,8 +98,8 @@ const create = context => {
 			const member = source.getText(node.property);
 
 			// Member might already be destructured
-			const destructuredMember = destructurings.find(property =>
-				property.key.name === member
+			const destructuredMember = destructurings.find(({key}) =>
```

Comment:
The problem with destructuring is that you lose some readability information. It's no longer clear when you read this that `key` is the key of a **property**...